vcx/wrappers/node/demo/faber-verify-signature.js

```diff
@@ -0,0 +1,74 @@
+import {CredentialDef} from "../dist/src/api/credential-def";
```

Comment:
These new demo files share common code with the classic credential exchange demo - lot's of code could be extracted and shared here, but I didn't want to do too many different things in this pull request. The point of demo here is really just to demonstrate the changes I made to `rustlib.ts`. Demo refactoring could be a separate pull request, I think.